src/app/modules/main/communities/minting/models/token_model.nim

```diff
@@ -30,14 +29,31 @@ QtObject:
     new(result, delete)
     result.setup
 
+  proc updateItem*(self: TokenModel, contractAddress: string, deployState: DeployState) =
```

Comment:
Maybe this should be named `updateDeployState` instead, since that's the only thing it updates

Reply:
Done